bazel/repository_locations.bzl

```diff
@@ -319,4 +319,14 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "kafka-python-2.0.0",
         urls = ["https://github.com/dpkp/kafka-python/archive/2.0.0.tar.gz"],
     ),
+    proxy_wasm_cpp_sdk = dict(
+        sha256 = "14f66f67e8f37ec81d28d7f5307be4407d206ac5f0daaf6d22fa5536797bcac1",
+        strip_prefix = "proxy-wasm-cpp-sdk-31f1fc5b7e09f231fa532d2d296e479a113c3e10",
+        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/archive/31f1fc5b7e09f231fa532d2d296e479a113c3e10.tar.gz"],
+    ),
+    proxy_wasm_cpp_host = dict(
+        sha256 = "d1fb98a40b38b99393b56b3509dac5f01a769512aebc86dcc37782ba29549517",
+        strip_prefix = "proxy-wasm-cpp-host-8c33ecf3784b00d0bc6b88bbe9de6d4e51239f88",
+        urls = ["https://github.com/proxy-wasm/proxy-wasm-cpp-host/archive/8c33ecf3784b00d0bc6b88bbe9de6d4e51239f88.tar.gz"],
+    ),
```

Comment:
What is the governance/review process for these repos? Who are owners, etc.? Do you have any documentation on how this split works in terms of what is in the external repos vs. what is inside this repo? Concretely in the near term I would like to make sure I am also reviewing PRs in the ^ repos until things stabilize?

Reply:
Since we just pushed the first version the OWNERS are just Piotr and myself.  When we get it moved to an organization we will adopt the rules of that organization.  In the meantime we can use the same sort of rules that Enovy uses and invite interested folks who are making contributions.

In terms of the division there is a C++ SDK repo, some of which is used here both to build the Wasm artifacts and to support the Null VM.   There is a C++ host repo which holds all of the non-host specific code.   I have made a first cut, but I am open to reorganizing the division between host-generic and host-specific.

In terms of the review of the non-Envoy proxy-Wasm code, we can create a new empty branch in each repo and create a PR adding the existing code to that branch and do the review there and then merge that branch back onto master.

Note that the Proxy-Wasm host code contains more than just the bits required by this Envoy PR.  I would suggest that rather than trying to shrink it down, we instead do it by subset of files.  There are some good logical separations: WasmVM, Wasm, Context, NullVm, V8, WAVM.

Reply:
https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/1 PTAL

Reply:
Thanks will do, on these reviews today. Agree on your plan of doing file subsets, etc. Happy to do whatever will work most efficiently.